### DISCUSSION TITLE
처음 시작해서 무작정 따라해보는 중인데, 점프가 잘 동작하지않아 질문드립니다.

### DISCUSSION THREAD

**Discussion by Manjok1** [naverz/zepeto-studio-kor#1097] [5. Q&A]
![10](https://user-images.githubusercontent.com/88345350/181455997-22914c76-c40c-43bc-9c0b-7e9470f537eb.gif)

따로 건드린 것 없이 가이드라인을 따랐습니다 (제가 판단하기로는요...)
지금 영상과 다르게 동작하는 것이 두 가지인데,

1. 큐브가 돌아가는 속도가 너무 빠른데 정상적인 것인지
2. 아바타의 점프가 첨부한 gif처럼 끊기면서 동작해서 발판에 올라갈 수 없는데 어떻게 해결해야할지 궁금합니다.

Saving Prefab to immutable folder is not allowed: Packages/zepeto.character/resources/zepeto/ZepetoBaseModel.prefab
UnityEditor.EditorApplication:Internal_CallGlobalEventHandler ()
아 그리고 이 오류도 뜨는데 동작에 문제가 생기지 않을지 궁금합니다!

감사합니다

**Comment by Manjok1** [comment]
https://drive.google.com/file/d/1TDFQtrGhvu31MHfb-siYkyMepIY0GTrz/view?usp=sharing

혹시 답변에 도움이 될까하여 프로젝트 파일 첨부합니다

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요~ 크리에이터님 반갑습니다 :) 
기본으로 들어있는 회전 스크립트는 프레임 단위로 동작하지 않기 때문에 컴퓨터 사양에 따라서 매우 빠르게 Update()루프가 돌 경우 같이 빨라질 수 있습니다. 

```typescript
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'
import {Time} from 'UnityEngine';

export default class SampleScript extends ZepetoScriptBehaviour {

    Start() {    
        console.log("Hello ZEPETO Script");
    }

    Update() {
        this.transform.Rotate(3*Time.deltaTime, 0, 0);
    }

}
```
위와 같이 Time.deltaTime을 곱해주면 일정한 속도로 동작할 것입니다.
import Time을 해주는 것 잊지 마세요.

그리고 사용하신 캐릭터가 모션 V2인데 JumpPower값을 인스펙터에서 지금보다 숫자를 크게 변경해보면 점프 높이가 높아집니다. (보내주신 프로젝트 기준으로는 10~15 정도가 적합할 것 같네요) 
Use Double Jump에 체크하신 후 에디터에서 shift키를 누르면 더 높이 점프할 수 있습니다.

감사합니다.

**Reply by Manjok1** [reply]
아 Player 세팅에서 자세히 설정할 수 있군요! 답변 감사합니다 ^^
